src/query/v2/requests.hpp

```diff
@@ -356,6 +356,7 @@ struct ScanVerticesRequest {
   // expression whose result is returned for every vertex
   std::optional<std::vector<std::string>> vertex_expressions;
   std::optional<size_t> batch_limit;
+  std::optional<std::vector<OrderBy>> order_bys;
```

Comment:
Is there any difference between an empty optional and a "filled" optional that contains an empty vector? If no, then I would just go with a vector.